vizro-core/changelog.d/new_fragment.md.j2

```diff
@@ -8,7 +8,7 @@ Uncomment the section that is right (remove the HTML comment wrapper).
 <!--
 ### {{ cat }}
 
```

Comment:
Another special treat for @petar-qb so he won't need to manually add `.` ever again!